botorch/optim/initializers.py

```diff
@@ -363,7 +375,9 @@ def gen_batch_initial_conditions(
                         X_rnd_nlzd = torch.rand(
                             n, q, bounds_cpu.shape[-1], dtype=bounds.dtype
                         )
-                    X_rnd = bounds_cpu[0] + (bounds_cpu[1] - bounds_cpu[0]) * X_rnd_nlzd
+                    X_rnd = unnormalize(
+                        X_rnd_nlzd, bounds, update_constant_bounds=False
```

Comment:
I suspect this is the fix
```suggestion
                        X_rnd_nlzd, bounds_cpu, update_constant_bounds=False
```